src/Security/Authorization/Core/src/AuthorizationPolicyBuilder.cs

```diff
@@ -99,7 +99,7 @@ public AuthorizationPolicyBuilder Combine(AuthorizationPolicy policy)
     /// that the current user has the specified claim and that the claim value must be one of the allowed values.
     /// </summary>
     /// <param name="claimType">The claim type required.</param>
-    /// <param name="allowedValues">Values the claim must process one or more of for evaluation to succeed.</param>
+    /// <param name="allowedValues">Values the claim must possess one or more of for evaluation to succeed.</param>
```

Comment:
```suggestion
    /// <param name="allowedValues">Optional list of claim values, which the claim must match.</param>
```
 This could use the same language as https://github.com/dotnet/aspnetcore/blob/main/src/Security/Authorization/Core/src/ClaimsAuthorizationRequirement.cs#L22. While accurate, possess isn’t a very commonly used word in doc comments.

Reply:
I find that wording ambiguous, the former explanation clearly says that the `claim` must only possess one of those `allowedValues` in order to succeed, the latter is not clear about it but reads more like all of them must match.

Reply:
We should be consistent and use the same wording as ClaimsAuthorizationRequirement, feel free to clarify both if you want to specify that it can match any value in AlloewdValues

Reply:
Optional list of claim values, which the claim must match on of the values.